cclib/parser/gaussianparser.py

```diff
@@ -2462,16 +2694,6 @@ def extract_charges_spins(line,prop):
             except:
                 pass
 
-        # Extract Rotational Constants
-        # Example:
-        # Rotational constants (GHZ):           3.13081     1.24272     0.88960
-        # OR for linear molecules:
-        # Rotational constants (GHZ): ************ 12.73690 12.73690
-        # Note: rotational constant will be converted to wavenumber units (1/cm) to standardize across parsers
-        if line[:28] == ' Rotational constants (GHZ):':
-            splits = line[28:].split()
-            self.append_attribute("rotconsts", [float(splits[i]) for i in (-3, -2, -1)])
-
```

Comment:
This should not be deleted correct?

Reply:
I added some new code to avoid problems in certain calcs. Check for this part:

```
        elif line.find('Rotational constants (GHZ):') > -1:
            try:
                roconst = [float(line.strip().replace(':', ' ').split()[3]),
                                float(line.strip().replace(':', ' ').split()[4]),
                                float(line.strip().replace(':', ' ').split()[5])]
            except ValueError:
                if line.find('********') > -1:
                    roconst = [float(line.strip().replace(':', ' ').split()[4]),
                                    float(line.strip().replace(':', ' ').split()[5])]
            self.append_attribute("rotconsts", roconst)
```

Reply:
I already sent @oliver-s-lee some files with those issues for the data folder

Reply:
#1270 is the issue I believe, I'll take a look now

Reply:
Ok so there is a genuine issue here that's @jvalegre is trying to resolve, I'll continue discussion above where the new code block is